system-tests/projects/e2e/cypress/e2e/spec_bridge.cy.ts

```diff
@@ -0,0 +1,20 @@
+it('visits foobar.com and types foobar inside an input', () => {
+  cy.visit('primary_origin.html')
+  cy.get('[data-cy="cross_origin_secondary_link"]').click()
+
+  cy.origin('http://foobar.com:4466', () => {
+    cy.get('[data-cy="text-input"]').type('foobar')
+  })
+
+  // ensure stability
+  cy.then(() => {
+    const specBridgeIframe: HTMLIFrameElement = window.top.document.querySelector('.spec-bridge-iframe')
+    const currentBody = window.top.document.querySelector('body')
+
+    // make sure the spec bridge overlays the reporter/AUT and is not off the screen
+    expect(specBridgeIframe.offsetTop).to.equal(currentBody.offsetTop)
+    expect(specBridgeIframe.offsetLeft).to.equal(currentBody.offsetLeft)
+    expect(specBridgeIframe.clientWidth).to.equal(currentBody.clientWidth)
+    expect(specBridgeIframe.clientHeight).to.equal(currentBody.clientHeight)
```

Comment:
I know this PR is fixing a performance issue, but I believe the original reason the spec bridge needed to have the same dimensions as the AUT is for the sake of screenshots having the correct dimensions. 

I wonder if there was a behavioral regression as well that this fixes, but it doesn't look like we have test coverage around screenshot dimensions for screenshots taken within cy.origin(), so it didn't result in a failing test. If so, I think asserting on the screenshot behavior would be a good idea. It would also be better since it would assert on the behavior instead of the implementation details.

Reply:
It's unlikely there was a behavioral regression since the size of the frame was still correct.

Reply:
I don't think a few system tests could hurt to verify the screenshot in case there are some weird layout issues that occur in future.

Reply:
@chrisbreiding how is something like 70e32513ae1a561f38d76e3c0e054b178f48cc33? It's based on the system tests that currently exist for snapshotting to just verify the dimensions of clipped, element, and full page snapshots.

Reply:
Looks good! Those are exactly what I had in mind. 

Like I said in my other comment, I think they should be moved into the driver's tests. They would fit in well in [origin/commands/screenshot.spec.ts](https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/integration/e2e/origin/commands/screenshot.spec.ts). You'll just need to copy over the `check:screenshot:size` task or share it some manner.

Reply:
> Looks good! Those are exactly what I had in mind.
> 
> Like I said in my other comment, I think they should be moved into the driver's tests. They would fit in well in [origin/commands/screenshot.spec.ts](https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/integration/e2e/origin/commands/screenshot.spec.ts). You'll just need to copy over the `check:screenshot:size` task or share it some manner.

@chrisbreiding I got everything moved over into the driver and now looks to be working properly.